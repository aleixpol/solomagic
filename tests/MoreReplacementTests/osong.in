\ref something0
\mt somethingsomething osom somethingsomething

\ref something1
\mt somethingsomething bosom somethingsomething

