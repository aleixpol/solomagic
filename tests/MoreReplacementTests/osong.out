\ref something0
\mt somethingsomething osom somethingsomething
\ma somethingsomething osong somethingsomething

\ref something1
\mt somethingsomething bosom somethingsomething
\ma somethingsomething bosom somethingsomething
